pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -213,7 +215,13 @@ func commonArgs(cluster string, cfg *config.Cluster, networkName string, nodeNam
 	return args, nil
 }
 
-func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, name string, args []string) ([]string, error) {
+func runArgsForNode(logger log.Logger, node *config.Node, clusterIPFamily config.ClusterIPFamily, name string, args []string) ([]string, error) {
+	varVolume := "type=volume,target=/var"
+	if n := os.Getenv("KIND_EXPERIMENTAL_DOCKER_VOLUME_OPTIONS"); n != "" {
```

Comment:
if you look in the recent history this was actually N volumes covering specific subsets of /var, we've temporarily reverted to one volume.
how is this handled if we switch back to N volumes ...?

Reply:
Good question! I see that happened in https://github.com/kubernetes-sigs/kind/commit/8201cd9e69c2a4909afb5d2fa1c10e737acf7363 . 
If you had to switch back to N volumes you could just remove this environment variable altogether. I think it being explicitly unsupported gives you the right to do that at any moment and nobody has the right to complain about it ;-)
Tbh personally I'm mostly interested in /var/lib/containerd specifically, so if it were ever removed I'd maybe reimplement it (using a different env var name to avoid confusion!) and send another PR.